shell/components/formatter/LiveDate.vue

```diff
@@ -109,10 +109,10 @@ export default {
 
       let label = diff.label;
 
-      if ( diff === 0 ) {
+      if ( diff.diff === 0 ) {
```

Comment:
this is a duesy... did something change or did we get this horribly wrong before this change?

Reply:
We got it horribly wrong before this change! If you look at that `diffFrom` function you can see it's returning an object with a `diff` key. This bug causes the table formatter to show '0 secs' instead of 'Just now' - I think we just never noticed because that's shown so briefly.
https://github.com/rancher/dashboard/blob/9268de26403418c17f75d792132addcbcad0ee74/shell/utils/time.js#L33-L40